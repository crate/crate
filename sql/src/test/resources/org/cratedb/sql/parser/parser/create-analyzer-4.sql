create analyzer a4 extends a3 with (
    token_filters with (
        mytokenfilter,
        blubb with (
            stopwords='bla, blubb'
        )
    ),
    tokenizer alphabet with (
        a='b',
        c='d'
    ),
    char_filters with (
        html_strip
    )

)