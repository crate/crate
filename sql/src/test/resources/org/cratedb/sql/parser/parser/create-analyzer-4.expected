org.cratedb.sql.parser.parser.CreateAnalyzerNode@20bdf463
name: a4
statementType: CREATE ANALYZER
extends: 	
	org.cratedb.sql.parser.parser.TableName@bf2
	a3
	
	org.cratedb.sql.parser.parser.AnalyzerElements@593b8970

	Tokenizer: 		
		org.cratedb.sql.parser.parser.NamedNodeWithOptionalProperties@5b4f1255
		alphabet
		Properties: 			
			org.cratedb.sql.parser.parser.GenericProperties@243cb948
			{c=value: d
			type: CHAR(1) NOT NULL
			, a=value: b
			type: CHAR(1) NOT NULL
			}
	Properties: 		
		org.cratedb.sql.parser.parser.GenericProperties@2f0e73ba
		{}
	TokenFilters: 		
		org.cratedb.sql.parser.parser.TokenFilterList@ea8388e

		[0]:		
		org.cratedb.sql.parser.parser.NamedNodeWithOptionalProperties@4618fa26
		mytokenfilter
		Properties: 			null		[1]:		
		org.cratedb.sql.parser.parser.NamedNodeWithOptionalProperties@11cef8be
		blubb
		Properties: 			
			org.cratedb.sql.parser.parser.GenericProperties@2b2c8b14
			{stopwords=value: bla, blubb
			type: CHAR(10) NOT NULL
			}
	CharFilters: 		
		org.cratedb.sql.parser.parser.CharFilterList@795ee430

		[0]:		
		org.cratedb.sql.parser.parser.NamedNodeWithOptionalProperties@44d74990
		html_strip
		Properties: 			null
